internal/pkg/pkiinit/option/optionHelpers.go

```diff
@@ -131,7 +134,10 @@ func getPkiCacheDirEnv() string {
 }
 
 func deploy(srcDir, destDir string) error {
-	return copyDir(srcDir, destDir)
+	if err := copyDir(srcDir, destDir); err != nil {
+		return err
+	}
+	return markComplete(destDir)
```

Comment:
deploy(...) is also called during other options, like import and cache. do we want to create a sentinel file in those cases as well?

Reply:
yes